keydata.go

```diff
@@ -255,14 +261,21 @@ type keyDigest struct {
 	Digest []byte  `json:"digest"`
 }
 
+type hkdfData struct {
+	Alg  hashAlg `json:"alg"`
+	Salt []byte  `json:"salt"`
+}
+
 type authorizedSnapModelsRaw struct {
 	Alg       hashAlg           `json:"alg"`
+	KDF       *hkdfData         `json:"kdf,omitempty"`
 	KeyDigest json.RawMessage   `json:"key_digest"`
```

Comment:
it might be a good time to add some comments about the fields' roles

Reply:
I've added some comments now.